var/spack/repos/builtin/packages/aml/package.py

```diff
@@ -31,7 +31,12 @@ class Aml(AutotoolsPackage):
     # version string is generated from git tags, requires entire repo
     version("master", branch="master", submodules=True, get_full_repo=True)
 
-    version("0.2.0", sha256="2044a2f3f1d7a19827dd9c0726172b690189b4d3fe938656c4160c022468cc4a")
+    version("0.2.1", sha256="bae49e89ed0f2a2ad3547430e79b7e4c018d6228c6ed951a12d59afd0b35f71c")
+    version(
```

Comment:
@perarnau is v0.2.0 being deprecated due to a CVE or specific issue?

Reply:
Severe performance bugs mostly. No CVE.

Reply:
Are we OK to mark this as resolved and proceed with merge?

Reply:
Unless the version is actually harmful to the user I'd rather just keep it as a previous version without the deprecated flag. Deprecation typically means that the version should be avoided at all costs and will actively bug users who elect to keep the deprecated version around on their computer.

Was the performance bug introduced in v0.2.0 or is it just a performance improvement in v0.2.1?